package.json

```diff
@@ -36,16 +36,16 @@
   "readme": "README.md",
   "dependencies": {
     "bluebird": "^3.5.5",
-    "cassandra-uuid": "^0.1.0",
     "content-type": "git+https://github.com/wikimedia/content-type#master",
     "entities": "^1.1.2",
     "fast-json-stable-stringify": "^2.0.0",
-    "hyperswitch": "^0.12.3",
+    "hyperswitch": "^0.12.4",
```

Comment:
This is not needed here

Reply:
I bumped the version of hyperswitch to 1.2.4 in the PR to indicate the change to uuid and code changes. Remaining at 0.12.3 would reintroduce cassandra-uuid.

Reply:
It's a bit confusing, I admit. The reason why the version shouldn't be bumped here is because we haven't changed RB's version in a while all the while introducing changes, so bumping the patch version here would be semantically wrong. On the other hand, we have the split coming up, and once that is merged we will create a new release, which will include this code.

Reply:
I assume this comment was in regards to the RB version number. I left all dependency version numbers the same except for service runner as requested